src/ngraph/runtime/cpu/builder/quantize.cpp

```diff
@@ -29,11 +29,11 @@ namespace ngraph
         namespace cpu
         {
             template <>
-            void Builder::BUILDER_DECL(ngraph::op::Quantize)
+            void Builder::BUILDER_DECL(ngraph::op::QuantizeCPU)
             {
                 if (runtime::cpu::mkldnn_utils::use_mkldnn_kernel(node))
                 {
-                    auto quantize = static_cast<const ngraph::op::Quantize*>(node);
+                    auto quantize = static_cast<const ngraph::op::QuantizeCPU*>(node);
```

Comment:
I dont see a reason why will be needing this op once we have ngraph ops ready. But IMO for time being we can have them like this. Maybe @jbobba can give us some more details on how to go about with this.

Reply:
Yes, let's take a future action to remove QuantizeCPU and use Quantize for the CPU backend.